app/viewmodels/checkAnswers/sections/guarantor/GuarantorErnVatSummary.scala

```diff
@@ -64,13 +66,29 @@ object GuarantorErnVatEoriSummary {
 
   private def getConsigneeSummary()(implicit request: DataRequest[_], messages: Messages): (String, String) =
     (
+      request.userAnswers.get(DestinationTypePage),
       request.userAnswers.get(ConsigneeExcisePage),
       request.userAnswers.get(ConsigneeExportVatPage)
     ) match {
-      case (Some(ern), _) =>
+
+      case (Some(TemporaryRegisteredConsignee), maybeErn, _) =>
+        "consigneeExcise.checkYourAnswersLabel.ernForTemporaryRegisteredConsignee" ->
+          maybeErn.map(HtmlFormat.escape(_).toString()).getOrElse(messages("guarantorErn.checkYourAnswers.notProvided", messages(s"guarantorArranger.$Consignee")))
+
+      case (Some(TemporaryCertifiedConsignee), maybeErn, _) =>
+        "consigneeExcise.checkYourAnswersLabel.ernForTemporaryCertifiedConsignee" ->
+          maybeErn.map(HtmlFormat.escape(_).toString()).getOrElse(messages("guarantorErn.checkYourAnswers.notProvided", messages(s"guarantorArranger.$Consignee")))
+
+      case (Some(ExportWithCustomsDeclarationLodgedInTheUk) | Some(ExportWithCustomsDeclarationLodgedInTheEu), _, maybeVat) =>
+        "consigneeExportVat.checkYourAnswersLabel" ->
+          maybeVat.map(HtmlFormat.escape(_).toString()).getOrElse(messages("guarantorErn.checkYourAnswers.notProvided", messages(s"guarantorArranger.$Consignee")))
+
+      case (_, Some(ern), _) =>
         "guarantorErn.checkYourAnswers.label" -> HtmlFormat.escape(ern).toString()
-      case (_, Some(vatNumber)) =>
+
+      case (_, _, Some(vatNumber)) =>
```

Comment:
Just a small thing, this case isn't touched by scoverage. Could a case be put in for this please?

Reply:
Additional commit to address that
<img width="920" alt="image" src="https://github.com/hmrc/emcs-tfe-create-movement-frontend/assets/11269301/cc530c99-6202-4f58-934e-7ff11d6e5857">